app/tests/api/api_v1/test_datasource.py

```diff
@@ -17,7 +17,22 @@ def test_read_data_sources(client: TestClient, db: Session) -> None:
 def test_return_200_for_existing_data_source(
     client: TestClient, data_source_id: int, db: Session
 ) -> None:
-    r = client.get(f"{settings.API_V1_STR}/data_source/{data_source_id}")
+    credentials = {
```

Comment:
This part is not needed. See my comment above on `/data_source/{data_source_id}` endpoint.

Reply:
okay